pkg/util/slice/slice_test.go

```diff
@@ -68,3 +68,12 @@ func TestShuffleStrings(t *testing.T) {
 		}
 	}
 }
+
+func TestSortInts64(t *testing.T) {
+	src := []int64{10, 1, 2, 3, 4, 5, 6}
+	expected := []int64{1, 2, 3, 4, 5, 6, 10}
+	Ints64(src)
+	if !reflect.DeepEqual(src, expected) {
```

Comment:
same issue, how is one supposed to guess, if Ints64(src) is meant to sort numbers.